src/bindings/js/node/tests/unit/infer_request.test.js

```diff
@@ -300,3 +300,47 @@ describe('ov.InferRequest tests', () => {
     });
   });
 });
+
+describe('ov.InferRequest tests with missing outputs names', () => {
+  const modelV3Small = testModels.modelV3Small;
+  let compiledModel = null;
+  let tensorData = null;
+  let tensor = null;
+  let inferRequest = null;
+
+  before(async () => {
+    await isModelAvailable(modelV3Small);
+
+    const fs = require('fs');
+    const core = new ov.Core();
+
+    let model_data = fs.readFileSync(getModelPath(modelV3Small).xml, 'utf8');
+    const weights = fs.readFileSync(getModelPath(modelV3Small).bin);
+    model_data = model_data.replace("names=\"MobilenetV3/Predictions/Softmax:0\"", "");
```

Comment:
On JS side we must using camelCase notation. Please, check that your **eslint** is active.